### PR TITLE
fix(speeddial): Drag and drop is not working on windows as expected when grid has more records

### PR DESCRIPTION
**Pull request by briyadav**
Drag and drop is not working on windows as expected when grid has more records
[https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-507225](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-507225)

### CODE REVIEW

File: packages/node_modules/@webex/widget-speed-dial/src/SpeedDialItem.tsx [hunk 1]
```diff
@@ -242,13 +242,13 @@ export const SpeedDialItem = forwardRef(({
...
+                        <path d="M30.3863 4.93905C30.1333 4.78333 29.8437 4.69717 29.5467 4.68927C29.2497 4.68137 28.9559 4.75201 28.695 4.89405L24.4395 7.59968V5.25C24.438 3.9078 23.9041 2.621 22.955 1.67192C22.006 0.722836 20.7192 0.188989 19.377 0.1875H5.87695C4.53474 0.188959 3.24792 0.722796 2.29884 1.67188C1.34975 2.62097 0.815912 3.90779 0.814453 5.25V18.75C0.815912 20.0922 1.34975 21.379 2.29884 22.3281C3.24792 23.2772 4.53474 23.811 5.87695 23.8125H19.377C20.7192 23.811 22.006 23.2772 22.955 22.3281C23.9041 21.379 24.438 20.0922 24.4395 18.75V16.3998L28.7455 19.1345C29.0028 19.2631 29.2886 19.3237 29.5759 19.3107C29.8632 19.2977 30.1425 19.2115 30.3871 19.0603C30.6317 18.9091 30.8336 18.6978 30.9736 18.4466C31.1137 18.1954 31.1872 17.9126 31.1872 17.625V6.375C31.1884 6.08699 31.1154 5.80353 30.9751 5.552C30.8348 5.30046 30.632 5.08935 30.3863 4.93905Z" fill="var(--mds-color-theme-text-primary-normal)" fillOpacity="0.95" />
```
Comment (Waseemraj) [inline packages/node_modules/@webex/widget-speed-dial/src/SpeedDialItem.tsx:251]: Is this change for fixing a dark theme issue?

**Review by Waseemraj — APPROVED** [review]

File: packages/node_modules/@webex/widget-speed-dial/src/SpeedDials.tsx [hunk 2]
```diff
@@ -179,13 +181,42 @@ export const SpeedDials = ({
...
+
+  const handleShouldCancelStart = (event: SortEvent | SortEventWithTag): boolean => {
+    // This should be the class name you use for your sortable items
+    const sortableItemClassName = 'wxc-speed-dial-item__drag-handle';  
+
+     // If the event is triggered by keyboard (e.g., spacebar), allow sorting to start
+     if (event.type === 'keydown') {
+      // Check if the correct key was pressed (e.g., spacebar)
+      if ((event as unknown as KeyboardEvent).key === ' ') {
+        console.log('spacebar pressed')
```
Comment (akulakum) [inline packages/node_modules/@webex/widget-speed-dial/src/SpeedDials.tsx:203]: Please remove the console log

**Review by akulakum — APPROVED** [review]

**Review by ravicra2 — APPROVED** [review]

File: packages/node_modules/@webex/widget-speed-dial/src/SpeedDials.tsx (hunk 2 above)
Comment (briyadav) [reply packages/node_modules/@webex/widget-speed-dial/src/SpeedDials.tsx]: Sure, Akula

File: packages/node_modules/@webex/widget-speed-dial/src/SpeedDialItem.tsx (hunk 1 above)
Comment (briyadav) [reply packages/node_modules/@webex/widget-speed-dial/src/SpeedDialItem.tsx:251]: Yes, Waseem

**Pull request merged**